### ISSUE TITLE
'Warning: A string ref, "VIEW", has been found within a strict mode tree. String refs are a source of potential bugs and should be avoided.

### ISSUE THREAD

**Issue by rsmelo92** [facebook/react-native#23132]
## Environment
```  
React Native Environment Info:
    System:
      OS: Linux 4.15 Ubuntu 18.04.1 LTS (Bionic Beaver)
      CPU: x64 Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
      Memory: 2.26 GB / 15.56 GB
      Shell: 4.4.19 - /bin/bash
    Binaries:
      Node: 8.3.0 - ~/.nvm/versions/node/v8.3.0/bin/node
      Yarn: 1.12.3 - /usr/bin/yarn
      npm: 5.3.0 - ~/.nvm/versions/node/v8.3.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      Android SDK:
        Build Tools: 26.0.3, 27.0.3, 28.0.3
        API Levels: 26, 27, 28
    npmPackages:
      react: 16.5.0 => 16.5.0 
      react-native: 0.57.0 => 0.57.0 
    npmGlobalPackages:
      react-native-cli: 2.0.1
```

## Description
Wrapped app with `<StricMode />` and after solving every alert, this one still appears

![image](https://user-images.githubusercontent.com/16295402/51639304-5589b680-1f3f-11e9-9171-ec8a3a109891.png)

after following the alert we found out that the error comes from this react native component:
https://github.com/facebook/react-native/blob/87b65339379362f9db77ae3f5c9fa8934da34b25/Libraries/Components/Keyboard/KeyboardAvoidingView.js#L55

------------------------------------

[StrictMode documentation click here](https://reactjs.org/docs/strict-mode.html)

**Comment by rsmelo92** [comment]
PR #23146 should fix it

**Comment by rsmelo92** [comment]
closed by #23146
